### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: clients/chain/chain_test.go [hunk 1]
```diff
@@ -6,49 +6,49 @@ import (
...
-	assert.Equal(t, Bsc.ElrondRoleProviderLogId(), "BscElrond-ElrondRoleProvider")
+func Test_multiversXRoleProviderLogId(t *testing.T) {
+	assert.Equal(t, Ethereum.MultiversXRoleProviderLogId(), "EthereumMultiversX-MultiversXRoleProvider")
+	assert.Equal(t, Bsc.MultiversXRoleProviderLogId(), "BscMultiversX-MultiversXRoleProvider")
 }
 
 func Test_ethRoleProviderLogId(t *testing.T) {
-	assert.Equal(t, Ethereum.EvmCompatibleChainRoleProviderLogId(), "EthereumElrond-EthereumRoleProvider")
-	assert.Equal(t, Bsc.EvmCompatibleChainRoleProviderLogId(), "BscElrond-BscRoleProvider")
+	assert.Equal(t, Ethereum.EvmCompatibleChainRoleProviderLogId(), "EthereumMultiversX-EthereumRoleProvider")
```
[the remark this answers is not in the document]
Comment (sstanculeanu) [reply clients/chain/chain_test.go:45]: indeed, updated

File: core/constants.go [hunk 2]
```diff
@@ -40,15 +40,15 @@ const (
 	// MetricLastEthereumClientError represents the metric used to store the last encountered error from the ethereum client
 	MetricLastEthereumClientError = "ethereum client last encountered error"
 
-	// MetricLastQueriedElrondBlockNumber represents the metric used to store the last elrond block number that was
-	// fetched from the elrond client
-	MetricLastQueriedElrondBlockNumber = "elrond last queried block number"
+	// MetricLastQueriedMultiversXBlockNumber represents the metric used to store the last MultiversX block number that was
```
[the remark this answers is not in the document]
Comment (sstanculeanu) [reply core/constants.go:43]: updated.. was not sure about them since these metrics are persistent

File: clients/multiversx/dataGetter.go [hunk 3]
```diff
@@ -39,22 +39,22 @@ const (
 type ArgsDataGetter struct {
 	MultisigContractAddress core.AddressHandler
 	RelayerAddress          core.AddressHandler
-	Proxy                   ElrondProxy
+	Proxy                   Proxy
 	Log                     logger.Logger
 }
 
-type elrondClientDataGetter struct {
+type multiversXClientDataGetter struct {
```
[the remark this answers is not in the document]
Comment (sstanculeanu) [reply clients/multiversx/dataGetter.go:46]: renamed

File: cmd/bridge/config/config.toml [hunk 4]
```diff
@@ -46,7 +46,6 @@
 
 [P2P]
     Port = "10010"
-    Seed = ""
```
Comment (dragos-rebegea) [inline cmd/bridge/config/config.toml:49]: Why is this line deleted ?

Comment (iulianpascalau) [reply cmd/bridge/config/config.toml:49]: in mx-chain-go, the rc/v1.4.0 got a new (better) method of generating deterministic p2p keys by using an external p2pKey.pem file

**Review by iulianpascalau — APPROVED** [review]

File: cmd/bridge/main.go [hunk 5]
```diff
@@ -393,11 +392,7 @@ func buildNetMessenger(cfg config.Config, marshalizer marshal.Marshalizer) (p2p.
 
 	p2pSingleSigner := &singlesig.Secp256k1Signer{}
 	p2pKeyGen := signing.NewKeyGenerator(secp256k1.NewSecp256k1())
-
-	privKey, _, err := crypto.CreateP2pKeyPair(cfg.MultiversX.PrivateKeyFile, p2pKeyGen, log)
-	if err != nil {
-		return nil, err
-	}
+	p2pPrivKey, _ := p2pKeyGen.GeneratePair()
```
Comment (iulianpascalau) [inline cmd/bridge/main.go:395]: :+1:

**Review by iulianpascalau — APPROVED** [review]

**Review by dragos-rebegea — DISMISSED** [review]

**Review by iulianpascalau — DISMISSED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]
